src/models/neural_networks/triplet_data.py

```diff
@@ -0,0 +1,235 @@
+from pathlib import Path
+import numpy as np
+import torch
+from torch.nn import functional as F
+from random import shuffle as shuffle_list
+
+from ..data import DataLoader, _BaseIter, ModelArrays, TrainData
+
+from typing import Union, Tuple, Optional, List, Iterable
+
+
+class TripletLoader(DataLoader):
+    def __init__(
+        self,
+        neighbouring_distance: Union[float, Tuple[float, float]] = 2,
```

Comment:
this is the `neighbourhood_size` parameter described in the tile2vec paper?

Reply:
Yup!

Reply:
have we already used that name or can we use it so that it's easier for people (future us) to go from paper to implementation?

Reply:
yea, I'll update this

Reply:
updated!